examples/podinfo-flux/git/podinfo-kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
Why are we making changes to the Flux API version in this PR?

Reply:
No reason, just moved it to it's own PR https://github.com/zarf-dev/zarf/pull/2830